[Refac] 스토리북 글로벌 스타일 설정

seokzin opened the pull request:
## 📌 이슈
- close #23 


<br/>

## 📝 설명

- `.storybook/` 내부 Alias 오류
  - 빌드 결과, 실제로 Alias는 잘 작동해요.
  - 다만 빨간 줄이 계속 뜨는 것은 번들러의 충돌로 추정은 해요.
  - 구글링을 1시간 동안 해봤는데 빨간 줄은 누구도 해결해주지 못했어요.. 그래서 상대경로로 다시 바꿨어요.

- 글로벌 스타일 적용
  - `addDecorator`를 통해 해결했어요.
  - 다만 config 파일에 jsx 문법이 들어가려면 확장자가 `.tsx`로 바뀌어야 해요. (TS라서 뜨는 오류)

<br/>

## 📷 스크린샷

![image](https://user-images.githubusercontent.com/43740455/156010887-31532423-fd02-4acd-b0d5-ca65e2066cf4.png)

![image](https://user-images.githubusercontent.com/43740455/156010771-4a089836-f5d7-4cfe-94fa-8c15c23a95d9.png)

seokzin committed:
feat: 스토리북에 Global Style 적용

seokzin committed:
fix: config 파일에 jsx 문법 사용을 위해 로 확장자 변경

devpla reviewed (approved):
이런! 너무 어렵네요. 고생했어요 🥺🥺

seokzin committed:
refac: Alias를 상대경로로 변경

- .storybook에서 Alias는 정상 작동해요.
- 다만 IDE에서 빨간 줄을 띄어주는데 이를 해결할 방법이 // @ts-ignore
  처리밖에 없어요.. (구글링만 1시간)
- 이 방법이 베스트는 아닌 것 같아서 다시 상대경로로 바꿨어요!
